### DISCUSSION TITLE
Instructor onboarding support

### DISCUSSION THREAD

**Discussion by karenword** [carpentries/trainers#119] [Local Communities]
How can local communities support Instructors/trainees through the checkout process, specifically? How can the community facilitate and encourage their decision to teach for the first time?

**Comment by karenword** [comment]
This question was addressed at a Trainers meeting on June 17, 2021. (We had less time for this question than for others.) This is a summary of meeting responses.

**Supporting checkout**
* offer local/custom checkout sessions (e.g. CarpentryConnect)
* offer practice demo sessions / counseling on checkout / "safe place to fail"
* Track checkout & follow up with those who haven't completed
* Check in with trainees
     * "guilt trip" with reminders about costs of training they received
     *  kindly ask where they are in the process and whether they need help
     * suggest other community participation meetings in the meantime

**Supporting decision to teach for the first time**
* Invite to teach a single episode
* Invite to join as a helper a couple of times before teaching 
* Maintain website (like "help wanted" page for lesson contributions) with upcoming opportunities to teach or observe, attend pre-workshop meetings to learn about planning etc.
* Offer gift cards/honoraria 


**Challenges**
* Short teaching opportunities create a lot of shift-changing and potential inconsistencies in a workshop
* People are really afraid of demos, especially being asked to start anywhere in a lesson.

**Comment by zpainter** [comment]
A great many of our instructors do this because they do not have many other places to teach, or to learn the skills they need to teach. That is something they can put in for their CVs, and it is a draw I recognize. I also find this is one of the few places that our instructors can get those skills too, as they might not have in a prior step of their career.

Some kind of way to play on that, or expand with another group if you have one (for us, perhaps the Office of Postdoctoral Affairs or someone in the School of Education) seems like something we could do more of to help potential new instructors feel good about their teaching ability.

**Comment by dvanic** [comment]
> How can local communities support Instructors/trainees through the checkout process, specifically? 

I think something helpful can also be for learners who are part of the instructor training together to agree to attend the same demo session for checkout - providing support, accountability and camaraderie in one go? This may be something we might want to suggest in Instructor Training.

> How can the community facilitate and encourage their decision to teach for the first time?

They need to be linked with their local community (even the online one, in times of COVID), so that they can be invited to take on a small role as part of an experienced teaching team. This can help learn the ropes before taking on bigger responsibilities.

**Reply by marklcrowe** [reply]
I'd like to help trainees through the checkout process, but one of the blockers I've encountered is that unless I am their workshop trainer and have their contact details through that, privacy issues make it very difficult for me to find out about and contact instructor trainees in my area. Even if I happen to learn that a trainee from one of my local institutions is on an instructor course (and a lot of time I wouldn't even hear that except by chance), I don't think the trainers on that course are allowed to give me their contact details to welcome them to our local community.
